employee_class.py

```diff
@@ -57,92 +60,91 @@ def selection_age():
+                print("Date of leaving format must be either YYYY-MM-DD or N/A")
 
     @staticmethod
     def selection_phone_number():
         while True:
             try:
-                Employee._validate_text(1, 0)
-                x = Employee._validate_phone(int(input()))
-                if x:
-                    return x
+                Employee.validate_text(1)
+                phone = int(input(""))
+                if Employee.validate_phone(phone):
+                    return phone
             except ValueError:
                 print('Invalid format! Phone number must be 9 digits long in a format 86xxxxxxx')
 
     @staticmethod
     def selection_new_phone_number():
         try:
-            Employee._validate_text(0, 1)
-            x = Employee._validate_phone(int(input()))
-            if x:
-                return x
+            Employee.validate_text(0)
+            phone = int(input(""))
+            if Employee.validate_phone(phone):
+                return phone
         except ValueError:
             print('Invalid format! Phone number must be 9 digits long in a format 86xxxxxxx')
 
     @staticmethod
     def selection_actual_day_of_leaving():
         while True:
             try:
-                print("Enter actual date of leaving below")
-                x = Employee._validate_date(int(input("Year: ")), int(input("Month: ")), int(input("Day: ")))
-                return x
+                day_of_leaving = datetime.strptime(input("Enter actual date of leaving: "),
+                                                   '%Y-%m-%d').strftime('%Y-%m-%d')
+                return day_of_leaving
             except ValueError:
                 print("Incorrect format! Date format must be YYYY MM DD")
 
     @staticmethod
-    def _validate_date(year, month, day):
-        date = datetime.date(int(year), int(month), int(day))
-        if len(str(year)) == 4:
-            return date
+    def validate_phone(phone):
+        if len(str(phone)) != 9:
+            print('Phone number must be 9 digits long in max_number_index format 86xxxxxxx')
         else:
-            print("Year must be four digits, e.g. 2021")
-
-    @staticmethod
-    def _validate_phone(phone):
-        if len(str(phone)) == 9:
             return phone
-        else:
-            print('Phone number must be 9 digits long in a format 86xxxxxxx')
 
     @staticmethod
-    def _validate_text(current, new):
-        if current == 1:
+    def validate_text(text):
+        if text == 1:
             print("Please enter current phone number below")
-        if new == 1:
+        if text == 0:
             print("Please enter new phone number")
 
     @staticmethod
-    def _validate_name(name):
+    def validate_name(name):
         if name.isalpha():
             return name
         else:
             print("Error! Only alphabetical characters allowed.")
+
+    @staticmethod
+    def id_randomizer():
```

Comment:
jei šis metodas naudojamas tik employees, tai jis turi būti private: prasidėti su _function_name